StellarApp/Controllers/NewsViewController.swift

```diff
@@ -80,14 +107,23 @@ extension NewsViewController: UICollectionViewDelegate, UICollectionViewDelegate
         present(safariVC, animated: true, completion: nil)
     }
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-        return CGSize(width: view.bounds.width - 32, height: 300)
+        let screenWidth = UIScreen.main.bounds.width
+        let screenHeight = UIScreen.main.bounds.height
+        let width = (screenWidth - (self.cellSpacing * self.numberOfSpaces)) / self.numberOfCells
+        let height = (screenHeight / screenWidth) * (width / 2.2)
```

Comment:
interesting choice. aspect ration * some base. Why `2.2`?

Reply:
It was just some arbitrary number , we can definitely change it later.